src/CarbonAware.CLI/src/Commands/Emissions/EmissionsCommand.cs

```diff
@@ -0,0 +1,57 @@
+using CarbonAware.Aggregators.CarbonAware;
+using CarbonAware.CLI.Common;
+using System.CommandLine;
+using System.CommandLine.Invocation;
+using System.Text.Json;
+
+namespace CarbonAware.CLI.Commands.Emissions;
+
+class EmissionsCommand : Command
+{
+    private readonly Option<string[]> _requiredLocation = CommonOptions.RequiredLocationOption;
+    private readonly Option<DateTimeOffset?> _startTime = new Option<DateTimeOffset?>(
+            new string[] { "--start-time", "-s" },
+            LocalizableStrings.StartTimeDescription)
+            {
+                IsRequired = false,
+                Arity = ArgumentArity.ExactlyOne,
+            };
+    private readonly Option<DateTimeOffset?> _endTime = new Option<DateTimeOffset?>(
+             new string[] { "--end-time", "-e" },
+            LocalizableStrings.EndTimeDescription)
+            {
+                IsRequired = false,
+                Arity = ArgumentArity.ExactlyOne,
+            };
+    public EmissionsCommand() : base("emissions", LocalizableStrings.EmissionsCommandDescription)
+    {
+        AddOption(_requiredLocation);
```

Comment:
This would be part of the common class.

Reply:
Noted as above: will likely defer until the next implementation as to whether or not to address.